### DISCUSSION TITLE
bone丢失

### DISCUSSION THREAD

**Discussion by jh521314** [blockscout/blockscout#9756] [Q&A]
我从shibarium转账bone到eth链，丢失了466个，超过24小时未到达。完全按照网站授权转移
![11](https://github.com/blockscout/blockscout/assets/117247642/3ccfcd19-35f4-4215-9548-f302c0f88be5)
![111](https://github.com/blockscout/blockscout/assets/117247642/27e4621e-829d-4a8a-af6e-7b7a466bf766)
![1111](https://github.com/blockscout/blockscout/assets/117247642/b3012144-e2ab-44db-8607-175ced005598)
希望官方人员能帮我查询并找回。

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
